helm-chart/images/binderhub/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
This useful version tag comes from the update to pip-tools 6.*, which also changes a couple of orderings below, but no actual change to resolution.